packages/react/src/components/Breadcrumb/index.test.tsx

```diff
@@ -1,60 +1,55 @@
-import React, { createRef } from 'react';
-import { shallow } from 'enzyme';
-import {
-  default as Breadcrumb,
-  BreadcrumbLink,
-  BreadcrumbItem
-} from 'src/components/Breadcrumb';
-import axe from '../../../axe';
+import React from 'react';
+import { render } from '@testing-library/react';
+import Breadcrumb, { BreadcrumbLink, BreadcrumbItem } from './index';
```

Comment:
Very very minor nit:

```suggestion
import Breadcrumb, { BreadcrumbLink, BreadcrumbItem } from './';
```